components/financial_assistance/app/models/financial_assistance/applicant.rb

```diff
@@ -1297,6 +1299,11 @@ def notify_of_eligibility_change
       # CoverageHousehold.update_eligibility_for_family(family)
     end
 
+    # Remove any evidences with nil attributes that may have been created
+    def remove_nil_evidences
```

Comment:
do we usually add a spec when we add a new method?

Reply:
> do we usually add a spec when we add a new method?

That line is being hit in `components/financial_assistance/spec/models/financial_assistance/application_spec.rb` which is on this branch